quilt/tools/main.py

```diff
@@ -139,7 +142,16 @@ def check_hash(group, hashstr):
     inspect_p.add_argument("package", type=str, help=HANDLE)
     inspect_p.set_defaults(func=command.inspect)
 
-    args = parser.parse_args()
+    return parser
+
+
+def main(args=None):
+    """Build and run parser
+
+    :param args: cli args from tests
+    """
+    parser = argument_parser()
+    args = parser.parse_args(args) if args is not None else parser.parse_args()
```

Comment:
I think `args = parser.parse_args(args)` should be ok - `None` is already the default value, so it just works.

Reply:
Not in the case where blank args are sent, if anyone ever tests what happens when `quilt` is called without args.  ..although, we may never test that.

Reply:
I misread what you said there.  Fixed.